man/src/container/exec.md

```diff
@@ -23,3 +23,36 @@ the same capabilities as the container, which may be limited. Set
    --user [user | user:group | uid | uid:gid | user:gid | uid:group ]
 
    Without this argument the command will be run as root in the container.
+
+# Exit Status
+
+The exit code from `docker exec` gives information about why the container
+failed to exec or why it exited.  When `docker exec` exits with a non-zero code,
+the exit codes follow the `chroot` standard, see below:
+
+**_125_** if the error is with Docker daemon **_itself_** 
+
+    $ docker exec --foo busybox; echo $?
+    # flag provided but not defined: --foo
+      See 'docker exec --help'.
+      125
```

Comment:
Isn't this example of an error within the Docker *CLI*, rather than the daemon, it's the CLI which isn't recognising the `--foo` here isn't it?

Reply:
Good question; will have to dig into the history of that. Perhaps this was done on purpose to being able to distinguish a `exit 1` (from the container's process) from an `exit 1` (from the docker cli (or daemon))?

Reply:
I can a different example if you guys have a common one off the top of your heads! I'm away for the weekend now, can try and recreate a 125 when I get back.

Reply:
not sure if another example is possible (as in; I think the `125` is generated in the CLI, as it won't send the flag to the daemon if it's invalid 🤔

Reply:
@thaJeztah IOW it is the description not the example which is wrong both here and in https://github.com/docker/cli/blob/561f6e399c3a0863c7b78ab797553b878cc4e73a/docs/reference/run.md#exit-status?

Reply:
Will I remove it for the moment?